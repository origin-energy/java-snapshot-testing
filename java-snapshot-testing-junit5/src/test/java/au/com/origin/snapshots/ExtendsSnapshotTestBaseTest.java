package au.com.origin.snapshots;

import org.junit.jupiter.api.Test;

public class ExtendsSnapshotTestBaseTest extends SnapshotTestBase {
  @Test
  void test() {
    snapshot("OK");
  }
}
