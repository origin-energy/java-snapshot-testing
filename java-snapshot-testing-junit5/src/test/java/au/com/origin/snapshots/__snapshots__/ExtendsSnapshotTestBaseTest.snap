au.com.origin.snapshots.ExtendsSnapshotTestBaseTest.test=[
OK
]